src/main/java/com/spotinst/sdkjava/model/api/admin/messageCenter/ApiGetMessage.java

```diff
@@ -0,0 +1,282 @@
+package com.spotinst.sdkjava.model.api.admin.messageCenter;
+
+import com.fasterxml.jackson.annotation.*;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+import com.spotinst.sdkjava.model.bl.admin.messageCenter.Metadata;
```

Comment:
Why bl class is included. This is api side?

Reply:
Unused import deleted